Userland/Libraries/LibCore/DirectoryEntry.cpp

```diff
@@ -36,11 +36,12 @@ static DirectoryEntry::Type directory_entry_type_from_posix(unsigned char dt_con
     VERIFY_NOT_REACHED();
 }
 
-DirectoryEntry DirectoryEntry::from_dirent(dirent const& de)
+ErrorOr<DirectoryEntry> DirectoryEntry::from_dirent(dirent const& de)
```

Comment:
Hm, not sure how I feel about changing this into `ErrorOr` just to be able to facilitate the temporary `DeprecatedString -> String` conversion. Maybe just use the `fixme` method instead for now?